docs/source/template_structure.html

```diff
@@ -61,28 +61,38 @@
 
 <div class="template_block">outputs
     <div class="template_block">output
-        <div class="template_block">execute_result</div>
+        <div class="template_block">output_area_prompt
+            <div class="template_block">execute_result</div>
+        </div>
```

Comment:
I don't think this is quite right:

1. The new block goes before the output content blocks (like `execute_result`), not wrapping them.
2. The new block is only in the HTML template; there's a separate section below for extra blocks in the HTML template.